frontend/src/Components/Link/index.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { useEffect, useContext } from "react";
+import { usePlaidLink } from "react-plaid-link";
+import Button from "plaid-threads/Button";
+
+import Context from "../../Context";
+
+interface Props {
+  currentPath: string;
```

Comment:
ah it appears this is not even used! that answers my earlier comments about this prop